docs/tutorial/sharing-our-app/index.md

```diff
@@ -58,7 +58,9 @@ an example command that you will need to run to push to this repo.
 Now that our image has been built and pushed into a registry, let's try running our app on a brand
 new instance that has never seen this container image! To do this, we will use Play with Docker.
 
-1. Open your browser to [Play with Docker](http://play-with-docker.com).
+1. Open your browser to [Play with Docker](https://www.docker.com/play-with-docker).
```

Comment:
If you prefer https only links, then https://labs.play-with-docker.com would also work. Maybe that's easier for users to follow the steps without scrolling and searching for the next link.